asteroid/engine/system.py

```diff
@@ -91,7 +93,10 @@ def common_step(self, batch, batch_nb, train=True):
         """
         inputs, targets = batch
         est_targets = self(inputs)
-        loss = self.loss_func(est_targets, targets)
+        if isinstance(self.loss_func, PITLossWrapper):
+            loss = self.loss_func(est_targets, targets, epoch=self.current_epoch)
+        else:
+            loss = self.loss_func(est_targets, targets)
```

Comment:
I don't think we'll accept that. 
Otherwise, we'll never stop growing the codebase for specific use cases.

Reply:
OK!
In the modified version (in another branch), I've tried using a callback class that updates the `beta` at the beginning of each epoch.

Reply:
Cool !
Feel free to open a PR with the other branch :smiley: